### DISCUSSION TITLE
How to determine if Infolist modal is open in table's `poll()` closure?

### DISCUSSION THREAD

**Discussion by plakhin** [filamentphp/filament#13391] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.92

### How can we help you?

I've noticed that in case I'm polling table content and Infolist is open, refresh causes Infolist reload and in case long Infolists it makes scroll jump to the top what is quite annoying. Actually I don't need to refresh the table while Infolist modal is open and my idea is to return `false` from the `poll()` call closure in that case, something like `->poll(fn (Infolist $infolist) => $infolist->isOpen ? false : '3s')`.

@danharrin Is there a way to determine if Infolist modal is open in the closure and will this do the trick?

**Comment by leandrocfe** [comment, marked as answer]
try

```php
ViewAction::make()
    ->mountUsing(function (Table $table) {
        $table->poll(null);
    })
```

**Comment by plakhin** [comment]
@leandrocfe this works just perfect! Thanks!
